src/termcolor/_types.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+Attribute = Literal[
+    "bold",
+    "dark",
```

Comment:
Rather than duplicating all these values from `termcolor.py`, can we re-use them?